PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Purpose
+_Describe the purpose of the pull request. Include background information if necessary._
+
+## Approach
+_How does this change fulfill the purpose?_
+
+#### TODOS and Open Questions
+- [ ] Use GitHub checklists. When solved, check the box and explain the answer.
+
+## Learning
+_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._
+
+## Screenshots
+_Let's see those sweet visuals!_
```

Comment:
i'm unsure why this file is in this PR, since it should be in master already as of a while ago, right?  did we lose it somewhere along the way?  maybe i'm confusing this with ui-eholdings...

Reply:
Its not in master as of this PR, yes, it is in ui-eholdings but missing from here.